tools/sof_tplg_fw_sanity_check/tplgfw_check.py

```diff
@@ -0,0 +1,739 @@
+#!/usr/bin/env python3
```

Comment:
I'd maybe reword "rise of closed source" and just have "With increasing number of processing components...".

Reply:
To be honest though it sounds like a great start to a horror tale for the linux mailing list